src/functions/helpers.js

```diff
@@ -54,6 +56,23 @@ export async function notifySlack(monitor, operational) {
   })
 }
 
+export async function notifyTelegram(monitor, operational) {
+  const text = `${operational ? '✅' : '❌'}
+  \`${monitor.method ? monitor.method : "GET"} ${monitor.url}\` - 👀 <${config.settings.url}|Status Page>
+  Monitor *${monitor.name}* changed status to *${getOperationalLabel(operational)}*`
+
+  const payload = new FormData()
+  payload.append('chat_id', TELEGRAM_CHAT_ID)
```

Comment:
`SECRET_` prefix is missing in your telegram secrets

Reply:
I guess that explains a lot.

Reply:
Fixed